examples/13-vrm/plot_vrm_fwd.py

```diff
@@ -0,0 +1,258 @@
+        titlestr12 = "True Model (y = 0 m)"
+        Ax12.set_title(titlestr12, fontsize=FS+2)
+
+        Cplot13 = mesh.plotSlice(
+            plotMap*xi_true, normal='Z', ind=int((ncz+2*npad)/2-1),
+            ax=Ax13, grid=True, pcolorOpts={'cmap': 'gist_heat_r'})
+        Cplot13[0].set_clim((0., np.max(xi_true)))
+        Ax13.set_xlabel('X [m]', fontsize=FS)
+        Ax13.set_ylabel('Y [m]', fontsize=FS, labelpad=-10)
+        Ax13.tick_params(labelsize=FS-2)
+        titlestr13 = "True Model (z = 0 m)"
+        Ax13.set_title(titlestr13, fontsize=FS+2)
+
+        norm = mpl.colors.Normalize(vmin=0., vmax=np.max(xi_true))
+        cbar14 = mpl.colorbar.ColorbarBase(
+            Ax14, cmap='gist_heat_r', norm=norm, orientation='vertical')
+        cbar14.set_label(
+            '$\Delta \chi /$ln$(\lambda_2 / \lambda_1 )$ [SI]',
+            rotation=270, labelpad=15, size=FS)
+
+        # PLOT DECAY
+        j1 = int((N**2-1)/2 - 3*N)
+        di_vrm = mkvc(np.abs(FieldsVRM[j1, :]))
+        di_tem = mkvc(np.abs(FieldsTEM[j1, :]))
+        Ax21.loglog(times, di_tem, 'r.-')
+        Ax21.loglog(times, di_vrm, 'b.-')
+        Ax21.loglog(times, di_tem+di_vrm, 'k.-')
+        Ax21.set_xlabel('t [s]', fontsize=FS)
+        Ax21.set_ylabel('|dBz/dt| [T/s]', fontsize=FS)
+        Ax21.tick_params(labelsize=FS-2)
+        Ax21.set_xbound(np.min(times), np.max(times))
+        Ax21.set_ybound(1.2*np.max(di_tem+di_vrm), 1e-5*np.max(di_tem+di_vrm))
+        titlestr21 = (
+            "Decay at X = " + '{:.2f}'.format(loc[j1, 0]) +
+            " m and Y = " + '{:.2f}'.format(loc[j1, 1]) + " m")
+        Ax21.set_title(titlestr21, fontsize=FS+2)
+        Ax21.text(1.2e-5, 18*np.max(di_tem)/1e5, "TEM", fontsize=FS, color='r')
+        Ax21.text(1.2e-5, 6*np.max(di_tem)/1e5, "VRM", fontsize=FS, color='b')
+        Ax21.text(1.2e-5, 2*np.max(di_tem)/1e5, "TEM + VRM", fontsize=FS, color='k')
+
+        j2 = int((N**2-1)/2 + 3*N)
+        di_vrm = mkvc(np.abs(FieldsVRM[j2, :]))
+        di_tem = mkvc(np.abs(FieldsTEM[j2, :]))
+        Ax22.loglog(times, di_tem, 'r.-')
+        Ax22.loglog(times, di_vrm, 'b.-')
+        Ax22.loglog(times, di_tem+di_vrm, 'k.-')
+        Ax22.set_xlabel('t [s]', fontsize=FS)
+        Ax22.axes.get_yaxis().set_visible(False)
+        Ax22.tick_params(labelsize=FS-2)
+        Ax22.set_xbound(np.min(times), np.max(times))
+        Ax22.set_ybound(1.2*np.max(di_tem+di_vrm), 1e-5*np.max(di_tem+di_vrm))
+        titlestr22 = (
+            "Decay at X = " + '{:.2f}'.format(loc[j2, 0]) +
+            " m and Y = " + '{:.2f}'.format(loc[j2, 1]) + " m")
+        Ax22.set_title(titlestr22, fontsize=FS+2)
+
+        # PLOT ANOMALIES
+        d2 = np.reshape(np.abs(FieldsTEM[:, 0]+FieldsVRM[:, 0]), (N, N))
+        d3 = np.reshape(np.abs(FieldsTEM[:, 10]+FieldsVRM[:, 10]), (N, N))
+        d4 = np.reshape(np.abs(FieldsTEM[:, 20]+FieldsVRM[:, 20]), (N, N))
+
+        Cplot31 = Ax31.contourf(x, y, d2.T, 40, cmap='magma_r')
```

Comment:
it looks like this plotting code is repeated a lot, could you turn it into a function
```
def plot_anomaly(data, title):
    .....
```